extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -735,6 +735,9 @@ export class AmpA4A extends AMP.BaseElement {
           const extensions = Services.extensionsFor(this.win);
           creativeMetaDataDef.customElementExtensions.forEach(
               extensionId => extensions.preloadExtension(extensionId));
+          // Preload any fonts.
+          (creativeMetaDataDef.customStylesheets || []).forEach(font =>
```

Comment:
Optional style nit: It's a good idea to use braces for imperative side-effecting callbacks, even if they're one-liners.